db/seeds.rb

```diff
@@ -22,5 +22,5 @@
 kudo3 = Kudo.create!(title: Faker::Food.dish, content: Faker::Food.description, employee_id: employee3.id, receiver_id: employee4.id, company_value: company_value1)
 kudo4 = Kudo.create!(title: Faker::Food.dish, content: Faker::Food.description, employee_id: employee4.id, receiver_id: employee3.id, company_value: company_value2)
 
-reward1 = Reward.create!(title: Faker::Food.dish, description: Faker::Food.description, price: 200)
-reward2 = Reward.create!(title: Faker::Food.dish, description: Faker::Food.description, price: 88)
+reward1 = Reward.create!(title: Faker::Food.dish, description: Faker::Food.description, price: 1)
```

Comment:
if you are not using `reward1` anywhere else you could call `Reward.create!' without assigning it to the local variable